### PR TITLE
Use the correct yard tag (@raise)

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

Comply with Yard syntax

## Was the API documentation (openapi.json) updated?
n/a

### CODE REVIEW

File: app/services/constituent_service.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def initialize(parent_druid:)
   # Typically this is only called one time (with a list of all the pids) because
   # subsequent calls will erase the previous changes.
   # @param [Array<String>] child_druids the identifiers of the child objects
-  # @raises ActiveFedora::RecordInvalid if AF object validations fail on #save!
+  # @raise ActiveFedora::RecordInvalid if AF object validations fail on #save!
   # @returns [NilClass, Hash] true if successful, hash of errors otherwise (if combinable validation fails)
```
Comment (mjgiarlo) [inline app/services/constituent_service.rb]: Should we also `s/@returns/@return/` or save that for another PR or meh?

Comment (jcoyne) [reply app/services/constituent_service.rb]: another pr please.

Comment (mjgiarlo) [reply app/services/constituent_service.rb]: @jcoyne :ok:

**Comment by ndushay** [comment]
oh no!  merge conflict!

**Pull request merged**